### PR TITLE
merge Api buildout/whats happening with main

### PR DESCRIPTION
**Pull request by Ryanjso**
Looking to merge the whats happening page to main

I looked it over on my phone and it looked good

screencapture-localhost-8000-whats-happening-2021-05-10-16_01_46
Screen Shot 2021-05-10 at 4 03 47 PM

https://www.figma.com/file/MrX0wdJXZ4OaRL9eDbDIo8/Client-Meeting-11-9-4-2021?node-id=1101%3A1557

(they know we weren't able to customize the calendar like the figma design)

### CODE REVIEW

**Comment by Ryanjso** [comment]
Yikes, it's failing cause it the tiny slider package for the carousel needs to access Window which you can't with SSR. I'm gonna try and see what I can do to fix it but I'm nervous 😩

**Comment by sreya-kata** [comment]
> Yikes, it's failing cause it the tiny slider package for the carousel needs to access Window which you can't with SSR. I'm gonna try and see what I can do to fix it but I'm nervous 😩

I tried to look up [solutions](https://www.gatsbyjs.com/docs/debugging-html-builds/) but they don't seem to work for me oof. If you can't fix it, I looked up other carousels and [react-multi-carousel](https://www.npmjs.com/package/react-multi-carousel) works on the server/netlify (and is pretty similar)! Here's a [preview ](https://deploy-preview-95--vital-site.netlify.app/) for it. [react-slick](https://react-slick.neostack.com/docs/get-started/) also works but it's less customizable

Also, in the production preview, the social media isn't rendering? Not sure why but I replaced the social media with <iframe> instead and that seemed to [work](https://deploy-preview-96--vital-site.netlify.app/whats-happening). 

You can look at the `Link-Components-Test` branch for both the things above^

**Comment by Ryanjso** [comment]
> react-multi-carousel

Ill check out that other one, cause I can't seem to fix this one for the life of me

THANK YOYU

Also mean't to add in my original comment, so all the pictures in the carousel would always be the same size I made them 16:9 ratio but that means the current placeholder one looks weird but they'll just have to remember in the future to make 16:9 graohics for that

File: src/components/carousel-shared/carousel.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* This file writes over css provided in the carousel package */
+
+.tns-nav {
+  display: flex;
+  justify-content: center;
+  z-index: 2;
+}
+
+/* .tns-nav > button {
```
Comment (josephmannis) [inline src/components/carousel-shared/carousel.css]: Can you remove this comment please?

**Review by josephmannis — APPROVED** [review]
Just one thing, looks good otherwise!

**Pull request merged**